notebooks/Data_Attribute_Recommendation_Regression_Model_Template.ipynb

```diff
@@ -0,0 +1,1933 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Attribute Recommendation - TechED 2020 INT260\n",
+    "\n",
+    "Getting started with the Python SDK for the Data Attribute Recommendation service."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Business Scenario\n",
+    "\n",
+    "We will consider a business scenario involving product master data. The creation and maintenance of this product master data requires the careful manual selection of the correct categories for a given product from a pre-defined hierarchy of product categories.\n",
```

Comment:
I think the introduction here still talks about the master data use case where we predict categories. Can you update that to reference the price?